cmd/root.go

```diff
@@ -109,7 +109,13 @@ func runE(v *viper.Viper, statz *stats.Stats, cmd *cobra.Command, args []string)
 
 	// find the config file if one was not specified
 	if configFile == "" {
-		if configFile, _, err = config.FindUp(workingDir, "treefmt.toml", ".treefmt.toml"); err != nil {
+		// look in PRJ_ROOT if set, otherwise fallback to the working directory
+		searchDir := os.Getenv("PRJ_ROOT")
+		if searchDir == "" {
+			searchDir = workingDir
+		}
+
+		if configFile, _, err = config.FindUp(searchDir, "treefmt.toml", ".treefmt.toml"); err != nil {
```

Comment:
I wonder if, if $PRJ_ROOT is set, the user intends to search from that directory or only look in that directory.

Reply:
Fair point. I've updated the logic and improved the test to check for this.